mlops/parallelm/mlops/constants.py

```diff
@@ -82,6 +82,14 @@ class HistogramType:
     TIMESTAMP_COLUMN = "time"
     HISTOGRAM_BIN_IDENTIFIER = " to "
 
+class MatrixType:
+    """
+    Constants used for matrix parsing
+    """
+    MATRIX_COLUMN = "matrix_columns"
```

Comment:
nit - given these names are used in to build the dataframe, do we need the prefix "matrix" for all? wouldnt columns, values and row_name suffice?